runtime/src/genesis_utils.rs

```diff
@@ -27,6 +27,7 @@ pub fn bootstrap_validator_stake_lamports() -> u64 {
 // Number of lamports automatically used for genesis accounts
 pub const fn genesis_sysvar_and_builtin_program_lamports() -> u64 {
     const NUM_BUILTIN_PROGRAMS: u64 = 4;
+    const NUM_PRECOMPILES: u64 = 2;
```

Comment:
Now that the feature ids passed to `Precompile::new` are both `None`, `Bank::finish_init` will always add the precompile accounts if they don't exist yet. This should be safe on live clusters because those precompile accounts already exist.

Reply:
seems so:

```
$ ./target/release/solana -ud account KeccakSecp256k11111111111111111111111111111

Public Key: KeccakSecp256k11111111111111111111111111111
Balance: 0.000000001 SOL
Owner: NativeLoader1111111111111111111111111111111
Executable: true
Rent Epoch: 0
Length: 17 (0x11) bytes
0000:   73 65 63 70  32 35 36 6b  31 5f 70 72  6f 67 72 61   secp256k1_progra
0010:   6d                                                   m

$ ./target/release/solana -ut account KeccakSecp256k11111111111111111111111111111

Public Key: KeccakSecp256k11111111111111111111111111111
Balance: 0.000000001 SOL
Owner: NativeLoader1111111111111111111111111111111
Executable: true
Rent Epoch: 0
Length: 17 (0x11) bytes
0000:   73 65 63 70  32 35 36 6b  31 5f 70 72  6f 67 72 61   secp256k1_progra
0010:   6d                                                   m

$ ./target/release/solana -um account KeccakSecp256k11111111111111111111111111111

Public Key: KeccakSecp256k11111111111111111111111111111
Balance: 0.000000001 SOL
Owner: NativeLoader1111111111111111111111111111111
Executable: true
Rent Epoch: 0
Length: 17 (0x11) bytes
0000:   73 65 63 70  32 35 36 6b  31 5f 70 72  6f 67 72 61   secp256k1_progra
0010:   6d                                                   m

$ ./target/release/solana -ud account Ed25519SigVerify111111111111111111111111111

Public Key: Ed25519SigVerify111111111111111111111111111
Balance: 0.000000001 SOL
Owner: NativeLoader1111111111111111111111111111111
Executable: true
Rent Epoch: 0
Length: 15 (0xf) bytes
0000:   65 64 32 35  35 31 39 5f  70 72 6f 67  72 61 6d      ed25519_program

$ ./target/release/solana -ut account Ed25519SigVerify111111111111111111111111111

Public Key: Ed25519SigVerify111111111111111111111111111
Balance: 0.000000001 SOL
Owner: 11111111111111111111111111111111
Executable: true
Rent Epoch: 0

$ ./target/release/solana -um account Ed25519SigVerify111111111111111111111111111

Public Key: Ed25519SigVerify111111111111111111111111111
Balance: 0.000000001 SOL
Owner: NativeLoader1111111111111111111111111111111
Executable: true
Rent Epoch: 0
Length: 15 (0xf) bytes
0000:   65 64 32 35  35 31 39 5f  70 72 6f 67  72 61 6d      ed25519_program

```